virtool/uploads/api.py

```diff
@@ -37,22 +38,17 @@ async def get(self) -> r200[List[GetUploadsResponse]]:
         Status Codes:
             200: Successful operation
         """
-        pg = self.request.app["pg"]
         user = self.request.query.get("user")
         upload_type = self.request.query.get("type")
 
         ready = get_req_bool(self.request, "ready")
 
-        uploads = await virtool.uploads.db.find(pg, user, upload_type, ready)
-
-        return json_response(
-            {
-                "documents": await apply_transforms(
-                    uploads, [AttachUserTransform(self.request.app["db"])]
-                )
-            }
+        uploads = await get_data_from_req(self.request).uploads.find(
+            user, upload_type, ready
         )
 
+        return json_response({"documents": uploads})
```

Comment:
The response doesn't match the method return value typing.

Create an `UploadSearchResult` model in core and import here. Return an `UploadSearchResult` object directly from the data layer method.